### DISCUSSION TITLE
How to update Tiddlywiki version on a TW file in Tidgi?

### DISCUSSION THREAD

**Discussion by Jeff314116** [tiddly-gittly/TidGi-Desktop#532] [Q&A]
Hi, I haven't found the way to udgrade the version of Tiddlywiki used by a TW file in Tidgi. Could you explain to me?

Thanks.
--
Jeff

**Comment by linonetwo** [comment]
Tiddlywiki is embeded in TidGI, so no way to update it unless install the latest version of TidGi.

Usually user don't know npm, so it is very hard for normal user to install tiddlywiki in terminal manually, so I didn't implement it.
